doc/radosgw/dynamicresharding.rst

```diff
@@ -48,6 +54,8 @@ Configuration
 .. confval:: rgw_dynamic_resharding
 .. confval:: rgw_max_objs_per_shard
 .. confval:: rgw_max_dynamic_shards
+.. confval:: rgw_dynamic_resharding_may_reduce
+.. confval:: rgw_dynamic_resharding_reduction_wait
```

Comment:
Shouldn't this PR add these to `src/common/options/rgw.yaml.in` or is that done elsewhere?

Reply:
added previously in https://github.com/ceph/ceph/pull/57538

Reply:
🙏🏼